{{cookiecutter.github_repo_name}}/Dockerfile

```diff
@@ -4,9 +4,14 @@ LABEL maintainer={{cookiecutter.git_email}}
 # Switch to super user to perform privileged operatiosn
 USER root
 
+# Change this value to regenerate the whole image
+ENV _UPDATED 2022-04-08
+
 # Install missing useful stuff
-RUN apt-get update
-RUN apt-get install -y git vim curl wget python3-wheel build-essential python3-dev libffi-dev
+RUN apt-get update && apt-get install -y git vim curl wget python3-wheel \
```

Comment:
python3-pip, xfonts-75dpi, xfonts-base, libssl-dev, zlib1g-dev sono già presenti nell'immagine base (o tirati dentro come dipendenze)
libsasl2-dev, libldap2-dev, libpq-dev, libxml2-dev e libxslt1-dev non sono strettamente necessarie, le ometterei. Mi affiderei piuttosto ad un meccanismo che permetta di installare pacchetti debian su un criterio repo-based, identico a requirements.txt, e che già abbiamo impostato con os_debian_requirements.txt di heiq. Meccanismo che non esiste ancora in ROCCO ma che se sei fortemente sul pezzo e carico sei libero di scrivere qui, e il mondo celebrerà le tue lodi.

Rationale di tutto ciò: essendo orientato ad ambienti dev, voglio evitare di semplificare la vita al dev inserendo cose che qui diamo per scontate ma che potrebbero non esserlo sui vari ambienti, col risultato che qui funziona, poi pushiamo e si rompe pure gesù.

Reply:
Makes sense.
Rimuovo le aggiunte ma mantengo l'update + install in una sola riga per ottimizzare il layer (tanto se installi prima devi fare update e se fai update probabilmente stai installando ...)